cockroachcloud/configure-scim-provisioning.md

```diff
@@ -0,0 +1,118 @@
+
+1. Log in to Okta Admin Dashboard as an admin user.
+1. Click **Applications** and edit the [SAML](configure-cloud-org-sso.html#saml) application for your {{ site.data.products.db }} organization.
+1. Click **Edit**.
+1. Click **Provisioning**.
+1. Select **SCIM** and click **Save**.
+1. In the integration's settings page, click **Provisioning** again, then click **Edit**.
+1. Click **Integrations**. This tab controls the SCIM app's authentication to the {{ site.data.products.db }} API. Set:
+
+    - **SCIM connector base URL**: `https://cockroachlabs.cloud/api/scim/v2`.
+    - **API authentication token**: the API token for a {{ site.data.products.db }} [service account](managing-access.html#create-a-service-account) with the [**Org Administrator (legacy)** role](authorization.html#org-administrator-legacy).
+    - **Unique identifier field for users**: `userName`
+    - **Authentication Mode**: **HTTP Header**
+
+1. Click **Test Connector Configuration**.
+1. Click **Save**.
+1. Click **To App**. This tab controls assignment of Okta identities to {{ site.data.products.db }}. To allow provisioning and deprovisioning of users, ensure that **Create Users** and **Deactivate Users** are selected, and make any other desired changes.
+1. Optionally, click **To Okta**. This tab allows you to perform a one-time import of a {{ site.data.products.db }} organization's existing users into Okta. Refer to Okta's documentation about mapping individual fields. Make any desired changes.
+
+To learn more, refer to [Add SCIM Provisioning to App Integrations](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SCIM.htm) in the Okta documentation
+
+## Assign a user
+
+To provision an Okta user to {{ site.data.products.db }}, assign the user to the SAML application.
+
+1. Log in to Okta Admin Dashboard as an admin user.
+1. Click **Applications** and click the [SAML](configure-cloud-org-sso.html#saml) application for your {{ site.data.products.db }} organization.
+1. Click **Assignments**. When an IdP user or group is assigned to an application, Okta allows them to sign in to the application.
+1. Click **Assign**, then select **Assign to People** or **Assign to Groups**.
+
+      Operations on an IdP group are applied to each member of the group individually by the {{ site.data.products.db }} API. For example, assigning an IdP group to the SCIM application allows the group's members to sign in to the application.
+
+1. Filter or search for a user or group.
+1. Next to them, click **Assign**, then **Save and go back**.
+
+      {{ site.data.products.db }} accounts are provisioned for IdP users who are assigned to the SAML application directly. If an IdP user is assigned to an application indirectly via membership in an IdP group, an account is provisioned only when the user first successfully authenticates to {{ site.data.products.db }} using the SAML application or when an IdP admin clicks **Push Groups**..
```

Comment:
Something along these lines "Users are auto-provisioned using assignment to the app integration. Users can also be provisioned via group membership if the group has been assigned. Push updates to groups alone will not provision new users in the app but will affect memberships of users that already exist in the app"